integration/server_test.go

```diff
@@ -1,71 +0,0 @@
-package docker
-
-import "testing"
-
-func TestCreateNumberHostname(t *testing.T) {
-	eng := NewTestEngine(t)
-	defer mkDaemonFromEngine(eng, t).Nuke()
-
-	config, _, _, err := parseRun([]string{"-h", "web.0", unitTestImageID, "echo test"})
-	if err != nil {
-		t.Fatal(err)
-	}
-
-	createTestContainer(eng, config, t)
-}
-
-func TestRunWithTooLowMemoryLimit(t *testing.T) {
```

Comment:
The entire server_test.go file has been removed by someone else before, and manual deletion here is just to solve my own git push issue.